new-lamassu-admin/src/pages/Commissions/helper.js

```diff
@@ -188,8 +246,8 @@ const defaults = {
 }
 
 const overridesDefaults = {
-  machine: '',
-  cryptoCurrencies: [],
+  machine: null,
+  cryptoCurrencies: '',
```

Comment:
Why did you need to change from empty array to ''? Isn't cryptoCurrencies a select multiple?